src/nfa/nfa.cc

```diff
@@ -542,7 +661,7 @@ void Mata::Nfa::make_complete(
         worklist.pop_front();
 
         std::set<Symbol> used_symbols;
-        if (!aut.has_no_transitions())
+        if (!aut.delta.has_no_transitions())
```

Comment:
could it be delta.empty() ?

Reply:
`empty()` has different semantics. `has_no_transitions()` is appropriate here. Technically, empty means that there are no states in `post`, however in current implementation you can have states in post without any transitions. Therefore we need `has_no_transitions()`.